Source/libsoldout/soldout_markdown.3

```diff
@@ -0,0 +1,373 @@
+.\"
+.\" Copyright (c) 2009 - 2016 Natacha Porté <natacha@instinctive.eu>
+.\"
+.\" Permission to use, copy, modify, and distribute this software for any
+.\" purpose with or without fee is hereby granted, provided that the above
+.\" copyright notice and this permission notice appear in all copies.
+.\"
+.\" THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+.\" WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+.\" MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+.\" ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+.\" WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+.\" ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+.\" OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+.\"
+.Dd May 30, 2016
+.Dt SOLDOUT_MARKDOWN 3
+.Os
+.Sh NAME
+.Nm soldout_markdown ,
+.Nm markdown
+.Nd parse markdown document
+.Sh SYNOPSIS
+.In markdown.h
+.Pp
+.Fd #define MKD_CELL_ALIGN_DEFAULT
+.Fd #define MKD_CELL_ALIGN_LEFT
+.Fd #define MKD_CELL_ALIGN_RIGHT
```

Comment:
this brings up an interesting conundrum: by using `ham` in your projects, you're supporting animal suffering (because, well, ham.). there should be a vegan alternative supported (perhaps even by default?)